dev-util/electron/electron-1.6.15.ebuild

```diff
@@ -746,11 +776,11 @@ src_compile() {
 	cd "${S}" || die
 
 	# Gather and prepare built components of libchromiumcontent.
+	create_dist_args+=" --target_arch=${target_arch} --component=static_library"
+	create_dist_args+=" --no_zip"
+	create_dist_args+=" --system-icu"
 	CHROMIUM_BUILD_DIR="${chromium_target}" \
-	python2 "${libcc_path}"/script/create-dist \
-		--target_arch=${target_arch} \
-		--component=static_library \
-		--no_zip || die
+	python2 "${libcc_path}"/script/create-dist ${create_dist_args} || die
```

Comment:
Please call `${EPYTHON}` here instead of `python2`, see https://wiki.gentoo.org/wiki/Project:Python/python-utils-r1#EPYTHON

Reply:
Must use python2 specifically here.  ${EPYTHON} may resolve to python3.  Is there a Gentoo-way of forcing a specific interpreter?

Reply:
> Is there a Gentoo-way of forcing a specific interpreter?

Yes, and the ebuild already uses it (python-any-r1.eclass).

Reply:
Sorry but the wording of the documentation is a bit sloppy and misleading, which is why you believe ${EPYTHON} may resolve to python3. I double checked with other devs and here's how the algorithm works out ${EPYTHON}:
* Iterate over the PYTHON_COMPAT array
* For each element found in the array
* Assign the value to ${EPYTHON}

Please give it a try and report back.

Reply:
Fixed!